src/FileUtility.ts

```diff
@@ -109,15 +109,15 @@ export class FileUtility {
   /**
    * Convert Json object to Json string and write into target file path.
    * @param type Scaffold type
-   * @param filePath tartet file path
+   * @param fileDestPath tartet file path
```

Comment:
It is ok to just use filePath as parameter name since function writeJsonFile indicate the filePath is target file path to write content.

Reply:
Yes. Just a enhancement to make it more apparent.